cloudant-client/src/main/java/com/cloudant/client/org/lightcouch/CouchDbException.java

```diff
@@ -52,6 +53,11 @@ public CouchDbException(String message, Throwable cause, int statusCode) {
         this.statusCode = statusCode;
     }
 
+    public CouchDbException(String message, Throwable cause, String url) {
```

Comment:
I guess it would be nice to do it the same way in both places, so I'm fine with this just calling the setter too.